app/src/androidTest/java/com/sdp13epfl2021/projmag/ProjectEditTest.kt

```diff
@@ -22,7 +22,7 @@ class ProjectEditTest {
         val intent = Intent(context, ProjectCreationActivity::class.java)
         val project = ImmutableProject(
             "a", "b", "c", "d", "e", "f", 4,
-            listOf(), false, true, listOf(), false, "a"
+            listOf(), false, true, listOf(), false, "a",authorToken = "token"
```

Comment:
avoid using string literals (since you are using the same value in many places, even if it is for tests). You could also define "token" as the default value for authorToken in ImmutableProject if you think this is a good idea.